### PR TITLE
Database Interface

### PR DESCRIPTION
**Pull request by saraedum**
as described in echemdb/echemdb#23.

Fixes echemdb/website#18.

Needs https://github.com/echemdb/svgdigitizer/pull/86 to be merged first so the tests pass.

### CODE REVIEW

File: echemdb/data/cv/database.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+r"""
+A Database of Cyclic Voltammograms.
```
[the remark this answers is not in the document]
Comment (DunklesArchipel) [reply echemdb/data/cv/database.py:2]: At least at the moment we only have CVs but I guess we don't want to be restrictive.

[the remark this answers is not in the document]
Comment (saraedum) [reply echemdb/data/cv/database.py:2]: Currently, this is in the `cv` namespace. So I think this is really only about CVs. Eventually, we want to move much of this code down one layer to `echemdb.data.database`.

File: templates/cv_entry.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Properties of the electrochemical cell:
+
+* Type: XXX
+* Counter electrode: XXX
+* Reference electrode: XXX
+
+<!-- TODO: Insert all the metadata from the .yaml file in some collapsible form here. E.g., just the YAML file with syntax highlighting. See #31. -->
+
+----
+
```
Comment (DunklesArchipel) [inline templates/cv_entry.md:32]:
```suggestion

<!-- TODO: Insert links to other data which are plotted in the same figure and/or even add a plot with all data from that figure. See #31 -->

```

Comment (DunklesArchipel) [reply templates/cv_entry.md:32]: would have to be added to #31 if we want this

**Review by DunklesArchipel — APPROVED** [review]
Aside from the small not super relevant remarks, I approve.

**Pull request merged**